### PR TITLE
Feature/voice js dns

### PR DESCRIPTION
**Pull request by charliesantos**
Merge feature

### CODE REVIEW

**Review by kbagchiGWC — COMMENTED** [review]
Wow more delete than add :)

File: lib/twilio/call.ts [hunk 1]
```diff
@@ -1860,11 +1855,6 @@ namespace Call {
      */
     reconnectToken?: string;
 
-    /**
-     * The Region currently connected to.
-     */
-    region?: string;
```
Comment (mhuynh5757) [inline lib/twilio/call.ts:1866]: Just making sure, we're OK with this API change for a minor version bump?

**Review by mhuynh5757 — APPROVED** [review]
Looks good to me, I don't immediately see any API changes, did I miss anything?

**Comment by mhuynh5757** [comment]
> Wow more delete than add :)

Mostly deprecated things :P I remember adding that years ago, going to miss it 😢 (jk)

**Comment by charliesantos** [comment]
> Looks good to me, I don't immediately see any API changes, did I miss anything?

Yep. No public API change. But if you look at region.ts, we are now directly using the new dns names. Before, we are mapping those to the old uris.

**Pull request merged**